app/views/NewExploreDeep/ProjectContent/TableView/index.tsx

```diff
@@ -0,0 +1,274 @@
+    TableHeaderCell,
+    createStringColumn,
+    createNumberColumn,
+    Pager,
+    useSortState,
+    SortContext,
+} from '@the-deep/deep-ui';
+
+import FrameworkImageButton, { Props as FrameworkImageButtonProps } from '#components/framework/FrameworkImageButton';
+import {
+    convertDateToIsoDateTime,
+    isFiltered,
+} from '#utils/common';
+import { createDateColumn } from '#components/tableHelpers';
+import { organizationTitleSelector } from '#components/selections/NewOrganizationSelectInput';
+import {
+    ProjectListQuery,
+    ProjectListQueryVariables,
+    ProjectOrderingEnum,
+} from '#generated/types';
+import { ORGANIZATION_FRAGMENT } from '#gqlFragments';
+
+import ActionCell, { Props as ActionCellProps } from '../ActionCell';
+import styles from './styles.css';
+
+const PROJECT_LIST = gql`
+    ${ORGANIZATION_FRAGMENT}
+    query ProjectList(
+        $search: String,
+        $organizations: [ID!],
+        $analysisFrameworks: [ID!],
+        $startDate: DateTime,
+        $endDate: DateTime,
+        $page: Int,
+        $pageSize: Int,
+        $ordering: [ProjectOrderingEnum!],
+        $regions: [ID!],
+    ) {
+        projects(
+            search: $search,
+            organizations: $organizations,
+            analysisFrameworks: $analysisFrameworks,
+            createdAtLte: $endDate,
+            createdAtGte: $startDate,
+            page: $page,
+            pageSize: $pageSize,
+            ordering: $ordering,
+            regions: $regions,
+        ) {
+            results {
+                id
+                title
+                createdAt
+                currentUserRole
+                regions {
+                    id
+                    title
+                }
+                analysisFramework {
+                    id
+                    title
+                }
+                stats {
+                    numberOfLeads
+                    numberOfUsers
+                }
+                isRejected
+                membershipPending
+                organizations {
+                    id
+                    organization {
+                        ...OrganizationGeneralResponse
+                    }
+                }
+            }
+            totalCount
+        }
+    }
+`;
+export type Project = NonNullable<NonNullable<NonNullable<ProjectListQuery['projects']>['results']>[number]>;
+
+const defaultSorting = {
+    name: 'CREATED_AT',
+    direction: 'Descending',
+};
+
+const projectKeySelector = (p: Project) => p.id;
+
+interface Props {
+    className?: string;
+    filters: ProjectListQueryVariables | undefined;
+}
+
+function ExploreDeepTableView(props: Props) {
+    const {
+        className,
+        filters,
+    } = props;
+
+    const [page, setPage] = useState<number>(1);
+    const [pageSize, setPageSize] = useState<number>(25);
+
+    useEffect(() => {
+        // NOTE: Bring user back to page 1 if filters have changed
+        setPage(1);
+    }, [filters]);
```

Comment:
It's best if we do this directly when filters change. This will cause on extra request to the server.